src/modules/callouts/pages/CalloutPage.vue

```diff
@@ -64,7 +64,33 @@
     <figure class="relative mb-6 pb-[56.25%]">
       <img class="absolute w-full h-full object-cover" :src="callout.image" />
     </figure>
-    <div class="text-lg content-message" v-html="callout.intro" />
+    <div
```

Comment:
I think it should always show the intro as a teaser. You can see similar behaviour for guests if you look at `<div v-if="showLoginPrompt">` code below